integrationtests/scenarios/i001641-rename-file-dependencies/before/paket.dependencies

```diff
@@ -0,0 +1 @@
+github forki/FsUnit:7623fc13439f0e60bd05c1ed3b5f6dcb937fe468 FsUnit.fs
```

Comment:
This paket.dependencies line will actively download a file during testing... That may be unwanted, if so: is there a work around to use local files?